CHANGELOG.md

```diff
@@ -4,6 +4,17 @@
 
 💥 Breaking changes:
 
+- Make components easier to link to from error summary
+
+  Updates the radios, checkboxes and date input components so it's easier to link to their first items from the error summary component.
+
+  To migrate:
+
+  You will need to ensure that your error summary still links correctly to your first input by removing any logic that adds a suffix to your id.
```

Comment:
I don't think this is very good, would welcome some ideas to make this better. Potentially some before and after examples?